docs/data/rpc/admin-guide.mdx

```diff
@@ -313,18 +324,19 @@ docker run -p 8001:8001 -p 8000:8000 \
 --config-path <RPC_CONFIG_FILENAME>
 ```
 
-You can use Stellar RPC itself to generate a starter configuration file:
+You can this [Stellar RPC subcommand](https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/README.md#configuring-and-running-rpc-server) to generate a starter configuration file:
 
 ```bash
-docker run stellar/stellar-rpc gen-config-file > soroban-rpc-config.toml
+docker run stellar/stellar-rpc:latest gen-config-file > soroban-rpc-config.toml
 ```
 
 The resulting configuration should look like this:
 
 ```toml
+
 # Admin endpoint to listen and serve on. WARNING: this should not be accessible
 # from the Internet and does not use TLS. "" (default) disables the admin server
-# ADMIN_ENDPOINT = "0.0.0.0:8001"
+# ADMIN_ENDPOINT = ""
```

Comment:
i'm assuming all these config file changes reflect the output you received from running that `docker run ... > soroban-rpc-config.toml` command, right? Nothing really to say about the changes, just checking.

Reply:
Yea I copy pasta'd the exact output from:
```
docker run stellar/stellar-rpc:latest gen-config-file > soroban-rpc-config.toml
```

The delta results from using the `latest` tag I'm guessing but yea that is the verbatim output from that command